scripts/generate_mesh_uniprot_mappings.py

```diff
@@ -1,30 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""Append lexical mappings between MeSH and UniProt."""
+
 import re
-import os
-from subprocess import check_output
-from indra.databases import mesh_client, hgnc_client
+from typing import Iterable, Tuple
+
+from indra.databases import hgnc_client, mesh_client
 
+from biomappings.resources import append_prediction_tuples
+from biomappings.utils import get_script_url
 
-def get_script_url():
-    commit_hash = check_output('git rev-parse HEAD'.split()).\
-        decode('utf-8').strip()[:6]
-    script_name = os.path.basename(__file__)
-    return (f'https://github.com/biomappings/biomappings/blob/{commit_hash}/'
-            f'scripts/{script_name}')
+MESH_PROTEIN_RE = re.compile(r'^(.+) protein, human$')
 
 
-def get_mappings():
-    url = get_script_url()
+def get_mappings() -> Iterable[Tuple[str, ...]]:
+    """Iterate high-confidence lexical mappings between MeSH and UniProt human proteins."""
+    url = get_script_url(__file__)
     mapping_type = 'lexical'
     match_type = 'skos:exactMatch'
+    confidence = 0.99
```

Comment:
This should be 0.999 :) I think 1 in 1000 error rate here is reasonable.

Reply:
I beat you to it with 50ea93e :)